internal/services/ai-bom-client/client_test.go

```diff
@@ -114,7 +113,7 @@ func TestGenerateAIBOM_Happy(t *testing.T) {
 		server.URL, // Use the test server URL
 	)
 
-	result, err := client.GenerateAIBOM(context.Background(), orgID, bundleHash)
+	result, err := client.GenerateAIBOM(t.Context(), orgID, bundleHash)
```

Comment:
The linter decided that these are now linting errors, I assume because of go version update